.github/settings.yml

```diff
@@ -160,7 +156,7 @@ labels:
 - name: dev/wont-fix
   color: fef2c0
   description: Will not fix the issue
-
+## The different sizes of an issue or pull request
```

Comment:
do we really need all these labels? i feel like they are a bit overkill

Reply:
depends... if we don't want to auto label PRs based upon their size, we don't need them... on the other hand we never played scrum poker and put our estimates on issues (yet)... should be a team decision

Reply:
The size depends on the changed LoC, right?
However, sometimes this metric can be confusing when a lot of LoC changes because of generate files, e.g. `go.mod` or `go.sum`.
Nevertheless, the label gives a short and quick assessment while looking on the issues on the projectboard. Thus, I would like to use them.

Reply:
Doing estimates is something, we should discuss in a retro as it is related to our general agile workflow.

I don't really see any benefit of labelling the PRs, because they are not displayed in the project board and if I am already on the PR I can get its size myself by clicking on `Files changed`.

However, when a PRs is connected to an issue, then we could (theoretically) auto-label the issue, right? If so, it could make sense to auto-label the issues to have a nice overview in the projectboard about all issues and their size.

What do you think?

Reply:
Should we keep the current setup and discuss this later after we tried out estimates?

Reply:
Let's go with these and continue I think. If it's too much we'll deal with it.